container_api/src/jsonrpc.rs

```diff
@@ -0,0 +1,49 @@
+use std::convert::TryFrom;
+
+use serde_json::{self, Value};
+
+use error::{HolochainInstanceError, HolochainResult};
+use holochain::Holochain;
+use holochain_core::{context::Context, logger::SimpleLogger, persister::SimplePersister};
+use holochain_core_types::{
+    cas::content::AddressableContent, error::HolochainError, json::JsonString,
+};
+
+use config::InterfaceConfiguration;
+
+#[derive(Serialize, Deserialize)]
+pub struct JsonRpcRequest {
+    pub jsonrpc: String,
+    pub method: String,
+    pub params: Value,
+    pub id: u32,
+}
+
+pub fn jsonrpc_success(id: u32, result: JsonString) -> JsonString {
+    json!({
+        "jsonrpc": "2.0",
+        // Question, is there a better way to safely go from JsonString into Value?
+        "result": serde_json::from_str::<Value>(&result.to_string()).unwrap(),
```

Comment:
Question for ya @thedavidmeister, can we make an `impl Into<serde_json::Value> for JsonString`? Or is there one I missed?